packages/lesswrong/components/comments/CommentsItem/CommentBody.tsx

```diff
@@ -55,6 +74,12 @@ const CommentBody = ({
   voteProps?: VotingProps<VoteableTypeClient>
   className?: string,
   classes: ClassesType,
+  potentialCommentsData?: {
+    potentialComments: {content: string, doppelId: string|null}[],
+    activePotentialComment: number,
+    setActivePotentialComment: React.Dispatch<SetStateAction<number>>,
+    doppelCommentVote: React.ReactNode,
+  },
```

Comment:
I think you're using this interface in multiple places, which justifies extracting it